fastplotlib/graphics/text.py

```diff
@@ -16,26 +16,6 @@ def __init__(
             outline_thickness=0,
             name: str = None,
     ):
-        """
```

Comment:
keep the docstring just don't work more on it, otherwise I will forget how this worked XD

Reply:
I made this docstring, there was no docstring before...do you still want me to leave it?